### DISCUSSION TITLE
Save costum attributes

### DISCUSSION THREAD

**Discussion by AlexVerm12** [SVG-Edit/svgedit#923] [Q&A]
Hi, 
I'm working on an extension for svg-edit to draw a seatplan. For that, i would like to give some costum attributes to the circle svg. Everything is fine, till I reload the site. After that, all costum attributes a cleared. Maybe someone can help to fix this.

this is how a give the circle element my attributes:

`newFO = svgCanvas.addSVGElementsFromJson({
                        element: 'circle',
                        attr: {
                            cx: opts.start_x,
                            cy: opts.start_y,
                            id: svgCanvas.getNextId(),
                            shape: 'circle',
                            r: 10,
                            radialshift: $id('radialShift').value,
                            fill,
                            stroke,
                            'stroke-width': strokeWidth,
                            'row-name': rowName[numberRows],
                            'seat-name': numberSeats,
                            available: 1,
                            'full-seat-name': rowName[numberRows] + '|' + numberSeats,
                            zone: '',
                            'seatplanseat_id': ""
                        }
                    })`

**Comment by shenggen1987** [comment]
you can see sanitize.js in program
